test/e2e_env/universal/meshaccesslog/meshaccesslog.go

```diff
@@ -209,6 +209,50 @@ spec:
 		Expect(dst).To(Equal("external"))
 	})
 
+	It("supports logging traffic to an ExternalService using MeshService", func() {
```

Comment:
With or without zone-egress? :)

Reply:
Are you saying I should add tests for both?

Reply:
I think it's likely just not working with zone-egress (I think we're missing plugin code to update the zone-egress config). What I was suggesting is having the name include whether or not that's with zone-egress